api/src/wfl/service/rawls.clj

```diff
@@ -37,7 +37,7 @@
   ([workspace snapshot-id name]
    (create-snapshot-reference workspace snapshot-id name ""))
   ([workspace snapshot-id]
-   (let [name (util/randomize (:name (datarepo/snapshot snapshot-id)))]
+   (let [name (:name (datarepo/snapshot snapshot-id))]
```

Comment:
Expects changes in @rexwangcc 's [PR](https://github.com/broadinstitute/wfl/pull/371) to name TDR snapshots something like:
```
<snapshotBaseName>_<YYYYMMDD>T<HHmmSS>_<shardIndex>
```

This serves as a realistically unique name.  Known possibility for collision: importing the same snapshot as a reference multiple times (we wouldn't do this).

Reply:
It would throw so this can't happen.

Reply:
Just a minor update, I corrected the timestamp to `<snapshotBaseName>_<YYYYMMdd>T<HHmmss>_<shardIndex>`.